src/backend/distributed/executor/multi_utility.c

```diff
@@ -1371,11 +1383,17 @@ PlanAlterTableStmt(AlterTableStmt *alterTableStatement, const char *alterTableCo
 static List *
 PlanRenameStmt(RenameStmt *renameStmt, const char *renameCommand)
 {
-	Oid relationId = InvalidOid;
+	Oid objRelationId = InvalidOid; /* SQL Object OID */
+	Oid tblRelationId = InvalidOid; /* Relation OID, maybe not the same. */
```

Comment:
For readability, we prefer full words over acronyms, i.e. objectRelationId, tableRelationId.

Reply:
this is changed now